test/Compiler/simple/ExportAndUse.out

```diff
@@ -1,6 +1,7 @@
 COMPILE_SUCCEEDED
 
 ret > ExitSuccess
+out > warning: -W[no]GenericWarning
```

Comment:
It feels like we should get rid of `GenericWarning`.
Could be that we need a warning mechanism for backends (`BackendWarning`?).
In this particular case, there is already a recipe for turning off the warning (`--no-main`), so the warning name should not be printed here.